docs/fsharp/style-guide/formatting.md

```diff
@@ -926,24 +1013,32 @@ let rainbow2 =
         Lackeys = [ "Zippy"; "George"; "Bungle" ] }
 ```
 
-You may want to dedicate separate lines for the braces and indent one scope to the right with the expression, however
-code formatters may reformat it. In some special cases, such as wrapping a value with an optional without parentheses, you may need to keep a brace on one line:
+You may want to dedicate separate lines for the braces and indent one scope to the right with the expression:
 
 ```fsharp
 // ✔️ OK
-let newState =
-    { state with
-          Foo = Some { F1 = 0; F2 = "" } }
-
-// ❌ Not OK, code formatters will reformat to the above by default
 let newState =
     {
         state with
             Foo =
-                Some {
-                    F1 = 0
-                    F2 = ""
-                }
+                Some 
+                    {
+                        F1 = 0
+                        F2 = ""
+                    }
+    }
+```
+
+You may also prefer the "Stroustrup" style of keeping the opening brace on the same line as the binding: 
+
+```fsharp
+let newState =
+    { state with
+        Foo =
+            Some { // TODO: Fantomas doesn't respect this yet
```

Comment:
The examples shouldn't mention what Fantomas does not respect (yet).
Once this is merged, we can follow up on Fantomas' side.

Reply:
Haha yeah anything marked `TODO` was mainly a note for myself haha, never planned to include those in the final draft. In this case I meant that as a note to myself to open an issue in fantomas if there isn't one already.